openfecwebapp/templates/partials/candidate/about-candidate.html

```diff
@@ -0,0 +1,145 @@
+{% import 'macros/missing.html' as missing %}
+{% import 'macros/null.html' as null %}
+{% import 'macros/cycle-select.html' as select %}
+
+<section id="section-2" role="tabpanel" aria-hidden="true" aria-labelledby="section-2-heading">
+  <div class="heading--section">
+    <h2 id="section-2-heading">About this candidate</h2>
+  </div>
+
+  <div class="slab slab--inline slab--neutral u-padding--left u-padding--right">
+
+    <div class="content__section">
+      {{ select.cycle_select(election_years, cycle, duration=duration, id="cycle-2") }}
+    </div>
+
+    <div class="entity__figure row">
+
+      <h3>Candidate information</h3>
+
+      <table class="t-sans usa-width-three-fourths">
+        <tr>
+          <td class="figure__label">Seat:</td>
+          <td class="figure__value">{{ constants.states[state] }} {{ office_full }}{% if office_full == 'House' %} district {{ district }}{% endif %}</td>
+        </tr>
+        <tr>
+          <td class="figure__label">Type of candidacy:</td>
+          <td class="figure__value">{{ incumbent_challenge_full }}</td>
+        </tr>
+        <tr>
+          <td class="figure__label">Political party:</td>
+          <td class="figure__value">{{ party_full|lower|title }}</td>
+        </tr>
+        {% if statement_of_candidacy %}
+        <tr>
+          <td class="figure__label">Statement of candidacy:</td>
+          <td class="figure__value">
+            <ul>
+            {% for statement in statement_of_candidacy %}
+              <li>
+                {% if loop.index == 1 %}
```

Comment:
It's already ordered by date, so the latest one will be the most recent.

In regards to showing all of the forms for the election, some of them have like 10+ revised statements of candidacy for an election and can really bog down that section.